pkg/skuba/actions/node/upgrade/apply.go

```diff
@@ -116,6 +117,13 @@ func Apply(client clientset.Interface, target *deployments.Target) error {
 		}
 	}
 
+	const drainTimeout = 15 * time.Minute
+	node := nodeVersionInfoUpdate.Current.Node
```

Comment:
I'm not sure if that is the right node to be referenced. :thinking:

Reply:
I would say is the correct one. I have doubts if this `kubernetes.GetNodeWithMachineID(client, target.Nodename)` would work, because I don't think `MachineID` is the same as `Nodename`.